docs/language/accounts.mdx

```diff
@@ -121,6 +124,11 @@ to the `prepare` phase of the transaction.
       fun getLinkTarget(_ path: CapabilityPath): Path?
       fun unlink(_ path: CapabilityPath)
 
+      // Storage iteration  
+      fun forEachPublic(_ function: ((PublicPath, Type): Bool)): Void
```

Comment:
If we don't make any guarantees about the order of iteration, is it actually useful to be able to terminate iteration using this boolean return value? What use cases would there be for wanting to get, say, 3 random NFTs out of storage without caring specifically which ones they are?

Reply:
Searching for specific `Type` can be some use case. Normally people will have one Collection Type in storage.